fastlane/lib/fastlane/actions/set_pod_key.rb

```diff
@@ -8,9 +8,9 @@ def self.run(params)
         cmd << ['bundle exec'] if File.exist?('Gemfile') && params[:use_bundle_exec]
         cmd << ['pod keys set']
 
-        cmd << ["\"#{params[:key]}\""]
-        cmd << ["\"#{params[:value]}\""]
-        cmd << ["\"#{params[:project]}\""] if params[:project]
+        cmd << ["\"#{params[:key].shellescape}\""]
```

Comment:
I'm not 100% certain, but I think when using `shellescape`, you shouldn't use the `"` around the string any more